reliability-engineering/terraform/modules/concourse-monitoring/files/dashboards/concourse.json

```
@@ -3247,7 +3098,10 @@
     "type": "timepicker"
   },
   "timezone": "browser",
-  "title": "Concourse",
+  "title": "Concourse old",
```

```suggestion
  "title": "Concourse",
```

The `old` one was the old dashboard moved out of the way to terraform this thing.